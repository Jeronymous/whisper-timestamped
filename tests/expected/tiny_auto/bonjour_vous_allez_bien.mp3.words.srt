1
00:00:00,460 --> 00:00:00,780
Боже

2
00:00:00,780 --> 00:00:02,260
улыл!

3
00:00:02,260 --> 00:00:02,360
Таков

4
00:00:02,360 --> 00:00:02,460
уже

5
00:00:02,460 --> 00:00:02,560
на

6
00:00:02,560 --> 00:00:03,300
меня!

7
00:00:33,000 --> 00:00:33,240
Боже

8
00:00:33,240 --> 00:00:33,800
улыл!

9
00:00:34,400 --> 00:00:34,800
Эскому

10
00:00:34,800 --> 00:00:34,960
за

11
00:00:34,960 --> 00:00:35,680
меня!

