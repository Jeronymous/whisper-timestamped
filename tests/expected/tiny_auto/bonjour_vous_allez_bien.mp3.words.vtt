WEBVTT

00:00.460 --> 00:00.780
Боже

00:00.780 --> 00:02.260
улыл!

00:02.260 --> 00:02.360
Таков

00:02.360 --> 00:02.460
уже

00:02.460 --> 00:02.560
на

00:02.560 --> 00:03.300
меня!

00:33.000 --> 00:33.240
Боже

00:33.240 --> 00:33.800
улыл!

00:34.400 --> 00:34.800
Эскому

00:34.800 --> 00:34.960
за

00:34.960 --> 00:35.680
меня!

