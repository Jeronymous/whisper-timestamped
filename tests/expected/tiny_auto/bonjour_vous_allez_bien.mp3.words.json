{
  "text": " Боже улыл! Эскому зарегиан! Боже улыл! Эскому зарегиан!",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.44,
      "end": 1.88,
      "text": " Боже улыл!",
      "tokens": [
        50364,
        5697,
        9292,
        1595,
        30975,
        693,
        0,
        50464
      ],
      "temperature": 0.0,
      "avg_logprob": -0.9443809107730263,
      "compression_ratio": 0.8166666666666667,
      "no_speech_prob": 0.04302893579006195,
      "confidence": 0.317,
      "words": [
        {
          "text": "Боже",
          "start": 0.44,
          "end": 0.86,
          "confidence": 0.423
        },
        {
          "text": "улыл!",
          "start": 0.86,
          "end": 1.88,
          "confidence": 0.261
        }
      ]
    },
    {
      "id": 1,
      "seek": 0,
      "start": 1.88,
      "end": 3.14,
      "text": " Эскому зарегиан!",
      "tokens": [
        50464,
        5381,
        4218,
        4161,
        17821,
        4953,
        435,
        1416,
        0,
        50564
      ],
      "temperature": 0.0,
      "avg_logprob": -0.9443809107730263,
      "compression_ratio": 0.8166666666666667,
      "no_speech_prob": 0.04302893579006195,
      "confidence": 0.395,
      "words": [
        {
          "text": "Эскому",
          "start": 1.88,
          "end": 2.3,
          "confidence": 0.286
        },
        {
          "text": "зарегиан!",
          "start": 2.3,
          "end": 3.14,
          "confidence": 0.505
        }
      ]
    },
    {
      "id": 2,
      "seek": 3000,
      "start": 32.98,
      "end": 34.02,
      "text": " Боже улыл!",
      "tokens": [
        50364,
        5697,
        9292,
        1595,
        30975,
        693,
        0,
        50564
      ],
      "temperature": 0.0,
      "avg_logprob": -0.3071819355613307,
      "compression_ratio": 0.8166666666666667,
      "no_speech_prob": 0.2232234627008438,
      "confidence": 0.577,
      "words": [
        {
          "text": "Боже",
          "start": 32.98,
          "end": 33.36,
          "confidence": 0.446
        },
        {
          "text": "улыл!",
          "start": 33.36,
          "end": 34.02,
          "confidence": 0.686
        }
      ]
    },
    {
      "id": 3,
      "seek": 3000,
      "start": 34.42,
      "end": 35.72,
      "text": " Эскому зарегиан!",
      "tokens": [
        50564,
        5381,
        4218,
        4161,
        17821,
        4953,
        435,
        1416,
        0,
        50664
      ],
      "temperature": 0.0,
      "avg_logprob": -0.3071819355613307,
      "compression_ratio": 0.8166666666666667,
      "no_speech_prob": 0.2232234627008438,
      "confidence": 0.849,
      "words": [
        {
          "text": "Эскому",
          "start": 34.42,
          "end": 34.84,
          "confidence": 0.691
        },
        {
          "text": "зарегиан!",
          "start": 34.84,
          "end": 35.72,
          "confidence": 0.992
        }
      ]
    }
  ],
  "language": "ru",
  "language_probs": {
    "en": 0.05109040066599846,
    "zh": 0.0006792841013520956,
    "de": 0.0007287654443643987,
    "es": 0.01429856475442648,
    "ru": 0.5119792819023132,
    "ko": 0.0008257994195446372,
    "fr": 0.021297650411725044,
    "ja": 0.001567091909237206,
    "pt": 0.2036512941122055,
    "tr": 0.0052192131988704205,
    "pl": 0.10648154467344284,
    "ca": 0.00012226558465044945,
    "nl": 0.008672517724335194,
    "ar": 0.0009284712723456323,
    "sv": 0.0007879833574406803,
    "it": 0.0003537872398737818,
    "id": 0.00035935858613811433,
    "hi": 3.0913583032088354e-05,
    "fi": 7.021122291916981e-05,
    "vi": 3.714352715178393e-05,
    "he": 0.00010457931057317182,
    "uk": 0.00486484169960022,
    "el": 0.00486484169960022,
    "ms": 0.0002489204052835703,
    "cs": 0.0018037117552012205,
    "ro": 0.0008388038841076195,
    "da": 1.0725262654887047e-05,
    "hu": 0.0007637396920472383,
    "ta": 3.6562209970725235e-06,
    "no": 0.0001316852285526693,
    "th": 1.203341184918827e-06,
    "ur": 0.00010581205424387008,
    "hr": 0.0008453827467747033,
    "bg": 0.007534814067184925,
    "lt": 0.00010581205424387008,
    "la": 0.002408271422609687,
    "mi": 6.268220658967039e-06,
    "ml": 4.929601345793344e-06,
    "cy": 0.000450736639322713,
    "sk": 0.0019200412789359689,
    "te": 1.409809283359209e-05,
    "fa": 6.318325904430822e-05,
    "lv": 9.672004671301693e-05,
    "bn": 2.209294143540319e-05,
    "sr": 0.00021542265312746167,
    "az": 5.341388168744743e-05,
    "sl": 0.0004969748551957309,
    "kn": 5.667552613886073e-07,
    "et": 3.4956110539496876e-06,
    "mk": 0.0002403213584329933,
    "br": 0.0026866206899285316,
    "eu": 4.677075776271522e-05,
    "is": 5.4680560424458236e-05,
    "hy": 4.1673874875414185e-06,
    "ne": 1.0363322644479922e-06,
    "mn": 9.958069313142914e-06,
    "bs": 0.00044374860590323806,
    "kk": 1.220088051923085e-05,
    "sq": 0.00021881508291698992,
    "sw": 1.3558002137870062e-05,
    "gl": 0.0006635484169237316,
    "mr": 8.262362030109216e-07,
    "pa": 1.8348761159359128e-06,
    "si": 0.00010457931057317182,
    "km": 0.00046869192738085985,
    "sn": 0.00016134412726387382,
    "yo": 0.00029444784740917385,
    "so": 4.1667655636956624e-07,
    "af": 1.9119846911053173e-05,
    "oc": 0.0002670526155270636,
    "ka": 6.517918791359989e-06,
    "be": 0.00028762698639184237,
    "tg": 3.018845688984584e-07,
    "sd": 1.2836458154197317e-05,
    "gu": 2.184769272162157e-07,
    "am": 1.0373449867984164e-06,
    "yi": 0.00014072682824917138,
    "lo": 9.43591373925301e-07,
    "uz": 1.9715000743047995e-09,
    "fo": 8.352842087333556e-06,
    "ht": 4.658841862692498e-05,
    "ps": 3.831686626654118e-06,
    "tk": 1.198291865733836e-08,
    "nn": 0.03350590169429779,
    "mt": 1.2393017641443294e-05,
    "sa": 6.752147601218894e-05,
    "lb": 8.849924171272505e-08,
    "my": 2.0073605355719337e-06,
    "bo": 4.703858849097742e-06,
    "tl": 7.429171546391444e-06,
    "mg": 8.09222822084621e-09,
    "as": 1.8674148805075674e-06,
    "tt": 9.030228511619498e-07,
    "haw": 0.0020279677119106054,
    "ln": 7.2146781349147204e-06,
    "ha": 3.7193925717105003e-09,
    "ba": 1.6799896584984708e-08,
    "jw": 0.0009730295278131962,
    "su": 1.2147872929801906e-08
  }
}